src/compiler/es-next/index.js

```diff
@@ -37,8 +37,14 @@ export default class ESNextCompiler {
         var presetES2015 = NODE_VER < 4 ? presetES2015Loose : presetES2015Node4;
 
         return {
-            presets:       [presetStage2, presetES2015],
-            plugins:       [transformRuntime],
+            passPerPreset: true,
```

Comment:
We need a comment with the link to the workaround description and testcafe issue